### PR TITLE
[sonos] Add support for Move 2

### LINKED ISSUE #15702: [sonos] Add support for Move 2
Logged by the binding during discovery:
```
2023-10-05 22:56:49.939 [WARN ] [overy.ZonePlayerDiscoveryParticipant] - 'Sonos Move 2' is not yet a supported model, thing type 'sonos:zoneplayer' is considered as default; please open an issue
```

## Your Environment
* Version used: 4.0

### PR DESCRIPTION
**Pull request by jlaur**
Resolves #15702

### CODE REVIEW

File: bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/Move2.xml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Sonos Move 2 Thing Type -->
+	<thing-type id="Move2" listed="false">
+		<label>Move 2</label>
+		<description>Represents SONOS Move 2 speaker</description>
+
+		<channels>
```
Comment (lolodomo) [inline bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/Move2.xml:12]: Please also add the channel "publicaddress" if the device has a line-input.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/Move2.xml (hunk 1 above)
Comment (jlaur) [reply bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/Move2.xml:12]: It has line-in, but unfortunately I cannot test it without an adapter. I have added the channels, thanks for noticing this.

I based the thing type on Move and tweaked it by adding the `linein` channel. Do you know if anything else needs to be done to have the correct set of channels?

Comment (lolodomo) [reply bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/Move2.xml:12]: > I based the thing type on Move and tweaked it by adding the `linein` channel. Do you know if anything else needs to be done to have the correct set of channels?

Most channels are common to all models. Only few of them are specific to certain models like battery channels for the models working on battery for example. Other example is the microphone channel.

I see you did not include bass and trebble channels. Are you sure they are not adjustable ? Please check if you can adjust them in the Sonos app.

Comment (jlaur) [reply bundles/org.openhab.binding.sonos/src/main/resources/OH-INF/thing/Move2.xml:12]: Indeed they are available, and `loudness` as well. I have added them and tested - they now sync correctly both ways. 🙂

**Review by lolodomo — APPROVED** [review]
LGTM, thank you

**Pull request merged**